assets/tests/CodeCheckerService.js

```diff
@@ -25,10 +25,18 @@ tie.factory('CodeCheckerService', [
       return code.indexOf(functionDefinition) !== -1;
     };
 
+    var _checkIfClassDefinitionExistsInCode = function(className, code) {
+      var classDefinition = 'class ' + className + '(object):';
```

Comment:
Hm, weird, it's not working. Lemme post a screenshot.

<img width="295" alt="screen shot 2017-04-17 at 2 22 50 pm" src="https://cloud.githubusercontent.com/assets/3400725/25105435/64251e0a-2379-11e7-994f-c8f8291473ca.png">